app/code/Codifi/CustomerRequest/view/frontend/layout/customer_account.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Codifi_CustomerRequest
+ *
+ * @copyright   Copyright (c) 2021 Codifi
+ * @author      Pavel Zelenevich <pzelenevich@codifi.me>
+ */
+-->
+<page xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" layout="2columns-left" xsi:noNamespaceSchemaLocation="urn:magento:framework:View/Layout/etc/page_configuration.xsd" label="Customer My Account (All Pages)" design_abstraction="custom">
+    <referenceBlock name="customer_account_navigation">
+        <block class="Magento\Customer\Block\Account\SortLinkInterface" name="customer-account-navigation-send-request-link" after="customer-account-navigation-newsletter-subscriptions-link">
+            <arguments>
+                <argument name="label" xsi:type="string" translate="true">Send Request</argument>
+                <argument name="path" xsi:type="string">customerrequest/account/index</argument>
```

Comment:
What the idea of naming controller 'Account' if no changes in account will happen? It's better to use 'customer/request/index' or 'customerrequest/form/index'